Multisig/UI/UI Library/Views/AddressInfoView.swift

```diff
@@ -45,16 +45,11 @@ class AddressInfoView: UINibView {
         titleLabel.isHidden = text == nil
     }
 
-    func setAddress(_ address: Address, label: String?, showIdenticon: Bool = true) {
+    func setAddress(_ address: Address, label: String?, imageUri: URL?, showIdenticon: Bool = true) {
```

Comment:
how about set the default value for imageURI = nil instead of passing it in many places

Reply:
done